SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.h

```diff
@@ -83,9 +84,12 @@ class Phase2TrackerDigitizerAlgorithm {
   void loadAccumulator(uint32_t detId, const std::map<int, float>& accumulator);
 
 protected:
-  // Accessing Lorentz angle from DB:
+  // Accessing Inner Tracker Lorentz angle from DB:
   edm::ESHandle<SiPixelLorentzAngle> SiPixelLorentzAngle_;
 
+  // Accessing Outer Tracker Lorentz angle from DB:
+  edm::ESHandle<SiPhase2OuterTrackerLorentzAngle> SiPhase2OTLorentzAngle_;
```

Comment:
member variables should start with lower case (does not seem to be followed in this class...)

Reply:
@suchandradutta please take note of this comment, I resolved a couple here.